plugins/Compressor/Compressor.cpp

```diff
@@ -211,7 +213,7 @@ void CompressorEffect::calcTiltCoeffs()
 {
 	m_tiltVal = m_compressorControls.m_tiltModel.value();
 
-	const float amp = 6.f / std::log(2.f);
+	const float amp = 6.f / std::numbers::ln2_v<float>;
```

Comment:
```suggestion
	constexpr float amp = 6.f / std::numbers::ln2_v<float>;
```
This could be constexpr now too